frontend/src/pages/Profile/UserSkills/SkillInfo.tsx

```diff
@@ -21,7 +21,7 @@ const CourseMaterialInfo: React.FC<Props> = ({
   <SkillInfo completed={stages === stagesCompleted}>
     <SkillInfoFlex>
       <SkillProgress>
-        <ProgressBar value={progress} size={SIZE.small} />
+        <ProgressBar value={progress} size={SIZE.small} variant="skills" />
```

Comment:
Would be better to create common constant VARIANTS and use it instead of strings everywhere